crates/anvil/src/eth/backend/executor.rs

```diff
@@ -1,15 +1,19 @@
 use crate::{
     eth::{
-        backend::{db::Db, validate::TransactionValidator},
+        backend::{
+            db::Db, mem::op_haltreason_to_instruction_result, validate::TransactionValidator,
+        },
         error::InvalidTransactionError,
         pool::transactions::PoolTransaction,
     },
-    inject_precompiles,
-    mem::inspector::Inspector,
+    // inject_precompiles,
```

Comment:
To do: restore precompile injection: https://github.com/foundry-rs/foundry/issues/10411